@plotly/dash-jupyterlab/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@plotly/dash-jupyterlab",
```

Comment:
It uses the tar bundle included in the Python package, that needs to be built the first time you open jupyterlab. Not sure if the npm package is used for anything, the jupyterlab docs says it can be directly to jupyterlab with npm but I am not sure the user really want to do that. https://jupyterlab.readthedocs.io/en/latest/extension/extension_tutorial.html#publishing-your-extension

I changed the name to avoid conflict if jupyter-dash is installed.